psalm.xml.dist

```diff
@@ -50,6 +50,11 @@
                 <referencedClass name="GdImage" />
             </errorLevel>
         </UndefinedClass>
+        <UndefinedAttributeClass>
+            <errorLevel type="suppress">
```

Comment:
should be added to baseline. reason is, that explicitly excluded issues are not (yet) reported by psalm once their unnecessary. 

Not a hard requirement but thats why I tend to put these to baseline

Reply:
Not baselined because result differs with php version psalm runs on.

Running on php 8.2 with --update-baseline removes baseline entry as unused which is how it got caught by CI here.
It still fails locally with error mentioned above but baselining that would be reported as unused in CI since it runs psalm on php 8.0.

Reply:
psalm is only executed on lowest locked deps, no clue why u have different errors at all unless dropping support for php versions or bumping psalm deps.

Reply:
Because locally psalm run on local php version

Reply:
TBH, the baseline is PHP version specific: I generate it with `docker run --rm -ti -v $(pwd):/project -w /project php:$VERSION` or such.

Reply:
To rephrase: agree with @boesing that this should be in baseline

Reply:
This **can't be in baseline** because as soon as contributor runs `composer static-analysis -- --update-baseline` on php 8.2 it will be dropped from baseline, because when psalm runs on 8.2 the reported issue is different.

Reply:
It **must** be in baseline, because:

 1. you run that command on the precise version you are running it on
 2. it should be disappearing the moment we drop PHP 8.1 support in the future